CONTRIBUTING.md

```diff
@@ -10,14 +10,36 @@ To discuss anything please [contact Cornelius](mailto:cschum@suse.de).
 
 ## Contribute code
 
-To contribute code please open pull requests.
+To contribute code fork the repository and open pull requests with your changes.
 
 Ensure that rspec and rubocop pass locally before sending your PR and always that you add new changes.
 
 If your changes include important new features or bug fixes please add them to the [Master (unreleased) section fo the CHANGELOG.md](https://github.com/openSUSE/trollolo/blob/master/CHANGELOG.md#master-unreleased)
 
+### Set up
+
+Clone git repo: 
+
+`git clone git@github.com:openSUSE/trollolo.git`
+
+Build the gem: 
+
+`gem build trollolo.gemspec`
```

Comment:
there is a rake task for that, take a look to the Rakefile 😉 This is actually worthwhile to mention 👍

Reply:
This is the rake task you need to run: `bundle exec rake gem:build`